core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -176,4 +176,16 @@ public boolean isReady() {
   public void setMessageCompression(boolean enabled) {
     // noop
   }
+
+  /**
+   * Sets the compression algorithm for this call.  If the server does not support the compression
+   * algorithm, the call will fail.
+   *
+   * @param compressor the name of the compressor to use.
```

Comment:
It isn't really clear the format/meaning of name. Providing an example would go along way.